deploy/helm-preview/services/workflow-dashboard/templates/service.yaml

```
@@ -0,0 +1,18 @@
+    labels:
+      app: {{ .Values.workflowsdashboard.nameOverride }}
+spec:
+    ports:
+    - name: {{ .Values.workflowsdashboard.nameOverride }}
+      port: {{ .Values.workflowsdashboard.service.port }}
+      protocol: {{ .Values.workflowsdashboard.service.protocol }}
+      targetPort: {{ .Values.workflowsdashboard.service.port }}
+    selector:
+        app: {{ .Values.workflowsdashboard.nameOverride }}
+    type: {{ .Values.workflowsdashboard.service.type }}
+{{- end }}
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)